### DISCUSSION TITLE
Welcome to awesome-test-automation Discussions!

### DISCUSSION THREAD

**Discussion by poliarush** [atinfo/awesome-test-automation#393] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by geokats7** [comment]
Hello everyone!
My name is Yorgos Katsaros and I'm into test automation, trying to build a robust solution for my company Orfium.
We have web applications and many microservices.
We use Python combined with different frameworks, related to the scope of the automation.
Our test runners are pytest, behave/pytest-bdd.
We also use selenium (with POM structure) and a custom containerized selenium grid on AWS for parallelization and cross-browser testing.
For API testing, we've started using the Tavern framework.

I joined this community with the hope to learn more and share ideas around test automation!
Cheers!
